README.adoc

```diff
@@ -226,6 +226,28 @@ With the experimental features you get a different output on the console, but yo
 
 NOTE: While these features are in the experimental phase, the options for switching buildkit on and off depend on the version of `docker` that you are using. Check the documentation for the version you have (the example above is correct for `docker` 18.0.6).
 
+== Security Aspects
+
+Just as in classic VM-deployments, processes should not be run with root permissions. Instead the image should contain a non-root user that runs the app.
+
+This can be achieved by adding another layer that adds a (system) user and group, then set it as the current user (instead of the default, root):
+
+`Dockerfile`
+[source]
+----
+FROM openjdk:8-jdk-alpine
+
+RUN groupadd -r mygroup && \
```

Comment:
I tried this, but the alpine image doesn't have a `groupadd`. Can you suggest something simple that works on alpine?